core/kits/documents/tabs/settings-site-identity.php

```diff
@@ -53,15 +53,15 @@ protected function register_tab_controls() {
 		$this->add_control(
 			$this->get_id() . '_refresh_notice',
 			[
-				'type' => Controls_Manager::RAW_HTML,
-				'raw' => sprintf(
+				'type' => Controls_Manager::ALERT,
+				'alert_type' => 'info',
```

Comment:
Can you also create constants for the `alert_type`?
```suggestion
				'alert_type' => Controls_Manager::ALERT_INFO
```

Reply:
We already introduced the alert control.

See https://developers.elementor.com/docs/editor-controls/control-alert/

This task is simply implementing the control in different panels.